workshops/templates/workshops/event.html

```diff
@@ -126,7 +126,7 @@ <h4 class="modal-title" id="preview_modal_label">Preview on Software-Carpentry w
 
 {% if tasks %}
 <h3>Tasks</h3>
-{% if all_emails %}<p><a href="mailto:{{ all_emails|join:',' }}" class="btn btn-primary">Mail everyone</a></p>{% endif %}
+{% if all_emails %}<p><a href="mailto:?bcc={{ all_emails|join:',' }}" class="btn btn-primary">Mail everyone</a></p>{% endif %}
```

Comment:
One question though: if this is using `?param=value` syntax, shouldn't the content of `value` be urlencoded?